# PathwayNet-Modelling branch

PathwayNet is a project based on Python and Bioinformatics. It tries to build up mathematic modes on simulating pathways of amino acid synthesis in *E. coli*. The simulation uses ODE numerical approach. By adjusting parameter of enzyme concentration and compounds' distrubution in reaction network, a higher engineering probiotics efficiency might be achieved. 

KEGG (Kyoto Encyclopedia of Genes and Genomes) will be the main databases for reference in pathways, compound structures and properties, and reaction mechanism. One cons about KEGG is that it does not provide reaction kinetics data, and all the kinetics will only be found out by the built math models. 

More details about modelling will be coming soon in this desctiptive documentation. 